chain.ts

```diff
@@ -5,7 +5,14 @@ import { ethers } from "ethers";
 import { getDai } from "./utils/getDai";
 
 const port = 8545;
+
+// Required environment variables.
 const nodeUrl = process.env.MAINNET_NODE_URL;
+if (!nodeUrl) {
```

Comment:
This might be overkill but it helps for user to know which env vars are required